Cargo.toml

```diff
@@ -22,6 +22,7 @@ members = [
     "components/support/rc_crypto/nss/nss_sys",
     "components/support/sql",
     "components/support/sync15-traits",
+    "components/storage-sync",
```

Comment:
nit: this is a *super* generic name which might cause confusion for newcomers to our repo, could we please consider calling this "extension-storage-sync" or "webext-storage-sync" or something?

(Obviously that's far from the most important issue facing the project, but also might be easier to address earlier rather than later).

Reply:
> could we please consider calling this "extension-storage-sync" or "webext-storage-sync" or something?

I'm thinking we should be optimistic, and assume we end up owning all webext storage, particularly for addons. Chome.Storage and Chrome.Storage.Sync have the same API, it's just that the latter syncs. So I'm thinking maybe just `webext-storage`?